document_retrieval/requirements.txt

```diff
@@ -24,3 +24,4 @@ urllib3==1.25.3
 Werkzeug==0.15.4
 wincertstore==0.2
 gensim==3.8.1
+waitress
```

Comment:
Do the requirement for `waitress` do not have a version specified?

To generate the `requirements.txt` file, use this:
```bash
pip freeze > requirements.txt
```
Should add the required module version as well.